solidjs-tailwind/src/components/PrivacyBadge/PrivacyBadge.jsx

```diff
@@ -1,12 +1,7 @@
-import { splitProps } from 'solid-js';
+import cn from 'classnames';
 
 const PrivacyBadge = (props) => {
-  const [local] = splitProps(props, ['visibility']);
-  return (
-    <span class="py-0.5 px-2 text-xs rounded-xl text-gray-600 border border-gray-300 font-medium capitalize">
-      {local.visibility.toLowerCase()}
-    </span>
-  );
+  return <span class={cn('py-0.5 px-2 text-xs rounded-xl text-gray-600 border border-gray-300 font-medium', props.class)}>{props.isPrivate ? 'Private' : 'Public'}</span>;
```

Comment:
the value here can't be as there are not just private and public.
return to the previous implementation. and probably in the query and API add visibility, you can use the user repo as a guide.

Reply:
The API returns `isPrivate`. So you can do:

`{local.isPrivate ? 'Private' : 'Public'}`